template.yaml

```diff
@@ -122,6 +122,14 @@ Resources:
             Action: sts:AssumeRole
       ManagedPolicyArns:
         - arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
+      Policies:
```

Comment:
You don't need to access policy to get tags since, the template turned metadata tags https://github.com/aws-samples/aws-plugin-for-slurm/blob/a4ccdab606d2cb90068995079a52418629c5723b/template.yaml#L192.

The reason for this is to avoid throttling on DescribeTags during large burst of instance creation.